tests/freqai/test_freqai_interface.py

```diff
@@ -287,6 +242,69 @@ def test_start_backtesting(mocker, freqai_conf, model, num_files, strat, caplog)
     shutil.rmtree(Path(freqai.dk.full_path))
 
 
+@pytest.mark.parametrize('model', [
+    'LightGBMClassifier',
+    'CatboostClassifier',
+    'XGBoostClassifier',
+    'XGBoostRFClassifier',
+    'SKLearnRandomForestClassifier',
+    'PyTorchMLPClassifier',
+    'TensorFlowClassifier',
+])
+def test_extract_data_and_train_model_Classifiers(mocker, freqai_conf, model):
```

Comment:
Is there a particular reason to change the sequence of these tests? (execution order in CI is randomized anyway, so the actual sequence does not matter).

at first glance, this seems to be the same test than the one we had (with additional lines) ... however moved around - which essentially makes a review more difficult (as it's no longer obvious what was actually changed).

Reply:
There's no particular reason. Prob got messed up while refactoring. I'll revert the chances and add the checks where necessary so we could test against it 👍🏼

Reply:
@xmatthias for some reason it shows that i added 152 lines to test_freqai_interface.py even though i haven't made any major changes. I only added the logic needed in  test_extract_data_and_train_model_Standard() and test_extract_data_and_train_model_Classifiers()

Reply:
I reverted the changes and added the tests where necessary.